.vscode/settings.json

```diff
@@ -1,5 +1,5 @@
 {
-    "editor.formatOnSave": true,
```

Comment:
Should this be part of the PR?

Reply:
`.vscode` related to local development, I don't understand why it's here? It would be better to remove at all.

Reply:
Can you remove it?